ql/math/optimization/levenbergmarquardt.hpp

```diff
@@ -55,23 +55,15 @@ namespace QuantLib {
         EndCriteria::Type minimize(Problem& P,
                                    const EndCriteria& endCriteria) override;
 
-        void fcn(int m,
-                 int n,
-                 Real* x,
-                 Real* fvec,
-                 int* iflag);
-        void jacFcn(int m,
-                 int n,
-                 Real* x,
-                 Real* fjac,
-                 int* iflag);
```

Comment:
They shouldn't have been in the public interface but they are.  I would deprecate them (and forward their calls to the private ones, of course).

Reply:
done